core/ArchiveProcessor.php

```diff
@@ -499,8 +504,15 @@ public function renameColumnsAfterAggregation(DataTable $table, $columnsToRename
             $columnsToRenameAfterAggregation = self::$columnsToRenameAfterAggregation;
         }
 
-        foreach ($columnsToRenameAfterAggregation as $oldName => $newName) {
-            $table->renameColumn($oldName, $newName);
```

Comment:
Before, we did iterate over each subtable for each columnToReplace over and over again. This is done in `table->renameColumn`. It is much faster to replace all columns at once and to iterate only once over all subtables.